boards/arm/nrf52840_pca10056/dts_fixup.h

```diff
@@ -0,0 +1,19 @@
+/* Board level DTS fixup file */
+
+#define DT_WNCM14A2A_UART_DRV_NAME			DT_NORDIC_NRF_UARTE_40028000_WNCM14A2A_BUS_NAME
```

Comment:
Why is this here?  We moved wcm14a2a out of the board.dts:

commit edefd7dc7d5f6b6eed1408d46fd53c0cacc01e69
Author: Kumar Gala <kumar.gala@linaro.org>
Date:   Thu Nov 15 09:58:49 2018 -0600

    boards: nrf52840_pca10056: move modem configurtion to overlay